recipe/meta.yaml

```diff
@@ -45,7 +53,11 @@ outputs:
   - name: flit
     version: {{ version }}
     build:
-      script: cd source/ && FLIT_ROOT_INSTALL=1 python -m flit install --deps=none
+      # there is an error with FLIT_ROOT_INSTALL on windows, the solution is to remove this section
+      # there is some information on this  on the following issue https://github.com/pypa/flit/issues/487
+      # in addition on this item: https://flit.pypa.io/en/latest/cmdline.html#envvar-FLIT_ROOT_INSTALL
```

Comment:
This comment, especially the link to the upstream ticket, is mis-leading. The issue is that `ENVVAR=VALUE some-command` is not valid syntax in Windows, and not that FLIT_ROOT_INSTALL is being stripped by `sudo`.

Reply:
The comments for the FLIT_ROOT_INSTALL were removed.

https://github.com/AnacondaRecipes/flit-feedstock/blob/d16c30ce386334db24089e5a7970ed67f92d1578/recipe/meta.yaml#L49&L54